launchable/test_runners/bazel.py

```diff
@@ -56,6 +58,35 @@ def f(case: TestCase, suite: TestSuite, report_file: str) -> TestPath:
 
     client.path_builder = f
     client.check_timestamp = False
-    client.scan(base, '**/test.xml')
+
+    if build_event_json:
+        lists = parse_build_event_json(build_event_json)
+        for l in lists:
+            path = Path(base).joinpath(l).resolve()
+            client.scan(path, "**/test.xml")
```

Comment:
Given that `path` here represents one Bazel package it seems odd to me that we are scanning `**/test.xml`.

Shouldn't it be just `client.report(Path(base).joinpath(l,'test.xml'))` ?

For that matter, for my own education, what is the point of `resolve()` call?

Reply:
> Shouldn't it be just client.report(Path(base).joinpath(l,'test.xml')) ?

yes right, so I fixed it. thank you

> For that matter, for my own education, what is the point of resolve() call?
In my understanding, `resolve()` resolves symlinks and eliminate “..” components.